pkg/vulnsrc/ghsa/ghsa.go

```diff
@@ -164,6 +167,21 @@ func (vs VulnSrc) commit(tx *bolt.Tx, ecosystem types.Ecosystem, entries []Entry
 		if err = vs.dbc.PutVulnerabilityID(tx, vulnID); err != nil {
 			return xerrors.Errorf("failed to save the vulnerability ID: %w", err)
 		}
+
+		// govuln doesn't have severity for vulnerabilities
+		// we use this details for this(for CVE-xxx and GHSA-xxx)
+		if ecosystem == vulnerability.Go && vulnID != entry.Advisory.GhsaId {
+			vulnID = entry.Advisory.GhsaId
+			vuln.ID = vulnID
+			if err = vs.dbc.PutVulnerabilityDetail(tx, vulnID, vulnerability.GHSA, vuln); err != nil {
+				return xerrors.Errorf("failed to save GHSA vulnerability detail: %w", err)
+			}
+
+			// for optimization
+			if err = vs.dbc.PutVulnerabilityID(tx, vulnID); err != nil {
+				return xerrors.Errorf("failed to save the vulnerability ID: %w", err)
+			}
+		}
```

Comment:
Why do we need this block? If I understand correctly, it satisfies the conditions of the if statement in the case of CVE-ID. Do we need to store GHSA-IDs having CVE-ID? I thought we wanted to store GHSA-IDs without CVE-ID.

Reply:
NVD has some delay in updating information about vulnerabilities. 
We are keeping  details for CVE-xxx from ghsa and will use them to get severity.

Reply:
Done